src/main/java/uk/gov/mca/beacons/api/services/DeleteBeaconService.java

```diff
@@ -53,6 +67,16 @@ public void delete(DeleteBeaconRequestDTO request) {
       .build();
     noteGateway.create(note);
 
-    beaconGateway.delete(request.getBeaconId());
+    // delete beacon owner
```

Comment:
@SianRMadeTech I would say we can remove these comments here and either:

- Extract the calls into their owns methods, i.e: `deleteBeaconOwner`, `deleteEmergencyContacts` etc...
- Or leave it as is

Typically the calls to the method on the class should give someone reading the code enough information to be able to determine what is happening.  So in this case does `ownerGateway.deleteBeaconId` describe that the owner is being deleted for a beacon.  I would argue yes but we could make it even more explicit by extracting this into a method on the class.  I will let you decide what you would prefer to do! 😄

Reply:
changes made